src/table/utils/handle-key-press.ts

```diff
@@ -54,22 +54,26 @@ export const getNextCellCoord = (
   evt: React.KeyboardEvent,
   rootElement: HTMLElement,
   cellCoord: [number, number],
-  topAllowedRow = 0
+  allowedRows: { cellIdx: number; totalsPosition: string } = { cellIdx: 0, totalsPosition: 'top' }
 ): [number, number] => {
   const rowCount = rootElement.getElementsByClassName('sn-table-row').length;
   const columnCount = rootElement.getElementsByClassName('sn-table-head-cell').length;
   let [nextRow, nextCol] = cellCoord;
 
   switch (evt.key) {
     case 'ArrowDown':
+      if (allowedRows.totalsPosition === 'bottom') {
+        nextRow < rowCount - 1 - allowedRows.cellIdx && nextRow++;
+        break;
+      }
       nextRow < rowCount - 1 && nextRow++;
       break;
     case 'ArrowUp':
-      nextRow > topAllowedRow && nextRow--;
+      nextRow > allowedRows.cellIdx && nextRow--;
       break;
     case 'ArrowRight':
-      // topAllowedRow greater than 0 means we are in selection mode
-      if (topAllowedRow > 0) break;
+      // allowedRows.cellIdx greater than 0 means we are in selection mode
+      if (allowedRows.cellIdx > 0) break;
       if (nextCol < columnCount - 1) {
         nextCol++;
       } else if (nextRow < rowCount - 1) {
```

Comment:
you probably need to add something here since you can press right arrow to go to next column

Reply:
the next row I mean of course

Reply:
Ah, you mean it should not move to Total row then, I thought it's how it works.. I compared it with the old table so I'll fix it.

Reply:
exactly. Yes this is a new feature, makes it a bit more accessible I think